generators/php/model/src/object/ObjectGenerator.ts

```diff
@@ -35,38 +35,35 @@ export class ObjectGenerator extends FileGenerator<PhpFile, ModelCustomConfigSch
             parentClassReference: this.context.getSerializableTypeClassReference()
         });
         // todo: handle extended properties
-        const requiredProperties = this.objectDeclaration.properties.filter(
```

Comment:
changes in this file are just using the new `getPropertyName` implementation, then some refactoring